pilot/pkg/config/memory/monitor.go

```diff
@@ -77,27 +80,32 @@ func newBufferedMonitor(store model.ConfigStore, bufferSize int, sync bool) Moni
 }
 
 func (m *configStoreMonitor) ScheduleProcessEvent(configEvent ConfigEvent) {
+	if m.closed.Load() {
```

Comment:
not process any event when the stop has been closed.

Reply:
lg